doc/Quick-Start-Guide.rst

```diff
@@ -14,7 +14,7 @@ The prerequisite that is necessary to install the Motr component is mentioned be
 
   - Install EPEL repo:
   
-    **$ sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm**
+    **$ sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm -y**
```

Comment:
This has been discussed in https://github.com/Seagate/cortx-motr/pull/468
This is not an automated script which must be run in non-interactive mode.
This is a guide for developer to prepare building environment.
So, asking for confirmation should not be an issue. Also, this step will have the developers to be noted that a thrid-party repo is going to be installed.

Reply:
Oh ok, I just thought it might be easier for the user to avoid having to be prompted. This is a good enough reason to leave it.